src/braket/aws/aws_session.py

```diff
@@ -60,8 +60,12 @@ def __init__(self, boto_session=None, braket_client=None, config=None, default_b
             )
             self.braket_client = braket_client
         else:
-            self.boto_session = boto_session or boto3.Session()
-            self.braket_client = self.boto_session.client("braket", config=self._config)
+            self.boto_session = boto_session or boto3.Session(
+                region_name=os.environ.get("BRAKET_REGION")
+            )
+            self.braket_client = self.boto_session.client(
+                "braket", config=self._config, endpoint_url=os.environ.get("BRAKET_ENDPOINT")
```

Comment:
nit: We may need to include docs for how a customer can find the Braket endpoints. A quick google search for Braket endpoints does yield https://docs.aws.amazon.com/braket/latest/developerguide/braket-regions.html as the top result so it isn't a difficult page to find.